packages/common/types/node-metamask/index.d.ts

```diff
@@ -33,3 +33,5 @@ declare module "node-metamask" {
     };
 
 }
+
+// test comment
```

Comment:
👎

Reply:
That was used to test changes on packages and be able to build a new image. Removed now after tests.